src/GitHub.Api/Git/Tasks/GitCommitTask.cs

```diff
@@ -8,17 +8,15 @@ class GitCommitTask : ProcessTask<string>
         private const string TaskName = "git commit";
         private readonly string arguments;
 
-        public GitCommitTask(string message, string body,
+        public GitCommitTask(string messageFile,
```

Comment:
This should probably be `NPath`, since that's what we have already from the temp file generation